parlai/core/torch_agent.py

```diff
@@ -321,14 +476,15 @@ def __init__(self, opt, shared=None):
         """Initialize agent."""
         super().__init__(opt, shared)
         opt = self.opt
-
         if not shared:
             # intitialize any important structures from scratch
             self.replies = {}  # past replies
             self.dict = self.dictionary_class()(opt)
             if opt.get('person_tokens'):
-                self.dict[self.P1_TOKEN] = 999999999
```

Comment:
???? This originally had the side effect of ADDING the tokens to the dictionary. Why add the guard?

Reply:
to make the test_torch_agent test pass

Reply:
could refactor `MockDictAgent` instead

Reply:
That seems more reasonable. The tests shouldn't dictate changes to the code.

Reply:
yep fixing